src/crafting.cpp

```diff
@@ -70,6 +70,8 @@ void load_recipe( JsonObject &jsobj, const std::string & /* src */, bool uncraft
     // required
     std::string result = jsobj.get_string( "result" );
     std::string category = uncraft ? "CC_NONCRAFT" : jsobj.get_string( "category" );
+    // @todo Fix the recipes and remove this
```

Comment:
It would be much bigger and require more testing.
I'd rather do it in a separate one, which could also include styling, normalization etc.
I don't have jq set up on Windows, meaning I couldn't properly whitelist the changed files.